packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1065,6 +1065,7 @@
   "Message_AllowDeleting": "Allow Message Deleting",
   "Message_AllowDeleting_BlockDeleteInMinutes": "Block Message Deleting After (n) Minutes",
   "Message_AllowDeleting_BlockDeleteInMinutes_Description": "Enter 0 to disable blocking.",
+  "Message_AllowDirectMessagesToYourself": "Allow user direct messages to yourself",
```

Comment:
I feel like this could be simply
> Allow direct messages to yourself

Reply:
Hey @gdelavald :-) I agree with you, although it was added so there was no doubt, since the phrase could be dubious to admins.